src/utils/DateUtils.ts

```diff
@@ -29,111 +130,174 @@ class DefaultDateToStringOptions extends Options implements IDateToStringOptions
   predefinedFormat: string = undefined;
 }
 
```

Comment:
This is where I would add the actual `DateUtils` class documentation:

> The `DateUtils` class exposes methods to convert strings, numbers, and Date objects to standard ISO 8601 Date objects, using the correct culture, language, and format. It also offers methods to convert Date objects to strings.

This is what I understand the class does. Please add anything else you feel is important to this description.